src/assets/javascript/components/buttom.js

```diff
@@ -0,0 +1,6 @@
+function buttom(textContent) {
+
+	return `<button>${textContent}</button>`;
```

Comment:
where are the class?

Reply:
I removed the class because i was trying to replace/remove the **"js-btn-pattern"** to other class of the different buttons.

Because when i was trying to change the class i was getting an error on the **addEventListener()**, 'cause when a removed/replaced the class **"js-btn-pattern"** it can't find the class.

without the class in the return of the **buttom function** i can just use a classList.add to add every class i want, is wrong doing by this way ??

Reply:
Yes, imagine that you have 1k class to handle event click, will you add 1k `case` into switch? it is a shit to maintain!
could be a good idea you create the button defining the class!

you should think how the button component can return the button html ready without need to change this dynamically!

other point is: when you is trying to replace html code, JS will search into the page, and if you have a lot of code to replace it gonna be slow and slow, make sense?

Reply:
yea,that's true. Before use `switch case` i used` if...else`, but it's the same work doing by this way.
i decided use switch, because i'm using simple `for loop` to print the buttons and add the events.

Reply:
Yeah I got it, But as I said

> you should think how the button component can return the button html ready without need to change this dynamically!